frontend/packages/dev-console/src/components/helm/HelmReleaseDetailsPage.tsx

```diff
@@ -1,144 +1,52 @@
 import * as React from 'react';
-import * as _ from 'lodash';
 import { match as RMatch } from 'react-router';
-import {
-  navFactory,
-  LoadingBox,
-  StatusBox,
-  FirehoseResult,
-} from '@console/internal/components/utils';
+import { Firehose, history } from '@console/internal/components/utils';
+import { ALL_NAMESPACES_KEY } from '@console/shared';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
 import { SecretModel } from '@console/internal/models';
-import { ErrorPage404 } from '@console/internal/components/error';
-import { DetailsPage } from '@console/internal/components/factory';
-import { K8sResourceKindReference } from '@console/internal/module/k8s';
-import { Status } from '@console/shared';
-import { Badge } from '@patternfly/react-core';
-import { fetchHelmReleases } from './helm-utils';
-import HelmReleaseResources from './HelmReleaseResources';
-import HelmReleaseOverview from './HelmReleaseOverview';
-import HelmReleaseHistory from './HelmReleaseHistory';
-import { deleteHelmRelease, upgradeHelmRelease } from '../../actions/modify-helm-release';
-import HelmReleaseNotes from './HelmReleaseNotes';
-import { HelmRelease } from './helm-types';
+import HelmReleaseDetails from './details/HelmReleaseDetails';
 
-const SecretReference: K8sResourceKindReference = 'Secret';
-const HelmReleaseReference = 'HelmRelease';
-export interface HelmReleaseDetailsPageProps {
+export const HELMRELEASES_ALL_NS_PAGE_URI = '/helm-releases/all-namespaces';
+export interface HelmReleaseDetailsProps {
```

Comment:
nit : we may not need `export` here

Reply:
and can we call it `HelmReleaseDetailsPageProps` as we have for other component any reason for having it as above